toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -380,20 +382,27 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 	}
 
 	// Customize the partitions.
-	partitionsCustomized, newRawImageFile, partIdToPartUuid, err := customizePartitions(ic.buildDirAbs,
+	partitionsCustomized, newRawImageFile, partIdToPartUuid, partUuidToMountPathFromPartitions, err := customizePartitions(ic.buildDirAbs,
 		ic.configPath, ic.config, ic.rawImageFile)
 	if err != nil {
 		return err
 	}
 	ic.rawImageFile = newRawImageFile
 
 	// Customize the raw image file.
-	err = customizeImageHelper(ic.buildDirAbs, ic.configPath, ic.config, ic.rawImageFile, ic.rpmsSources,
+	partUuidToMountPathFromExistingImage, err := customizeImageHelper(ic.buildDirAbs, ic.configPath, ic.config, ic.rawImageFile, ic.rpmsSources,
 		ic.useBaseImageRpmRepos, partitionsCustomized, ic.imageUuidStr)
 	if err != nil {
 		return err
 	}
 
+	// Determine which mountpoint mapping to use for COSI.
+	if ic.config.CustomizePartitions() {
+		ic.partUuidToMountPath = partUuidToMountPathFromPartitions
+	} else {
+		ic.partUuidToMountPath = partUuidToMountPathFromExistingImage
```

Comment:
`partUuidToMountPathFromExistingImage` and `partUuidToMountPathFromPartitions` are probably the same. So, you could drop the `partUuidToMountPathFromPartitions` and then just always use `partUuidToMountPathFromExistingImage`.